### PR TITLE
Update the test documentation on how to generate and use a JWT test token. 

### LINKED ISSUE #204: Update the test documentation on how to generate and use a JWT test token.
Update the test documentation on how to generate and use a JWT test token. 

/test/readme.md

### PR DESCRIPTION
**Pull request by Aymalla**
closes #204

### CODE REVIEW

File: data-reconciliation-app/Makefile [hunk 1]
```diff
@@ -80,7 +80,7 @@ clean: ## 🧹 Clean the working folders created during build/demo
 	@rm -rf dist
 
 # Generate a MS-Idp AAD (Azure active directory) JWT (app-to-app) token for the application testing;
-generate-jwt-token: # Generate an Azure AD access tokens for authentication
+generate-jwt-token-ms: # Generate an Azure AD access tokens for authentication
```
Comment (ross-p-smith) [inline data-reconciliation-app/Makefile]: I've moved this in my PR! Sorry!

File: data-reconciliation-app/test/README.md [hunk 2]
```diff
@@ -74,3 +74,24 @@ make test                      # Run end-to-end(e2e) tests in a sandbox (virtual
 **To Run the application's e2e-tests on a Managed CCF**
   - First, create a Managed CCF instance on your Azure subscription. Please follow [here](https://github.com/microsoft/ccf-app-samples/tree/main/deploy#deploying-the-ccf-samples)
   - Run the e2e-test, please follow [here](https://github.com/microsoft/ccf-app-samples/tree/main/deploy#deploying-a-ccf-application-to-azure-managed-ccf)
+
+## Test JWT tokens
+
+The application is currently supporting two JWT token issuers (identity providers):
```
Comment (ross-p-smith) [inline data-reconciliation-app/test/README.md:80]:
```suggestion
The application currently supports two JWT token issuers (identity providers):
```

File: data-reconciliation-app/test/README.md [hunk 3]
```diff
@@ -74,3 +74,24 @@ make test                      # Run end-to-end(e2e) tests in a sandbox (virtual
...
+- **Test Idp:** it's a custom implementation to simulate a testing token issuer to test your application locally.
```
Comment (ross-p-smith) [inline data-reconciliation-app/test/README.md:81]:
```suggestion
- **Test Idp:** custom implementation to simulate a token issuer to test your application locally.
```

**Review by macromania — APPROVED** [review]

File: data-reconciliation-app/Makefile (hunk 1 above)
Comment (Aymalla) [reply data-reconciliation-app/Makefile]: I've merged your changes to my branch and fixed this one

File: data-reconciliation-app/test/README.md [hunk 4]
```diff
@@ -74,3 +74,19 @@ make test                      # Run end-to-end(e2e) tests in a sandbox (virtual
...
+
+The application currently supports two JWT token issuers (identity providers):
+- **Test Idp:** custom implementation to simulate a token issuer to test your application locally.
+  - *Generate test tokens*: `/.workspace/proposals/set_jwt_issuer_test_sandbox.json` contains pre-generated tokens you can use to test the application endpoints using JWT authentication.
+  - Run ` make start-host` and request `/app/swagger` endpoint then `authorize`
+- **Microsoft Azure Active Directory Identity Provider:** it is an integration sample with MS-AAD Idp
+  - *Generate test tokens*: before you can complete this step, two applications must be registered at the Azure AD tenant, follow [here](https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) 
+  
+    - To create the required registered applications run `make deploy-ms-idp`.
+    - Update src/services/authentication-service.ts with environment values in `aad.env file` created by the previous step:
```
Comment (ross-p-smith) [inline data-reconciliation-app/test/README.md:88]:
```suggestion
    - Update `src/services/authentication-service.ts` with environment values in `aad.env` file created by the previous step: 
```

File: data-reconciliation-app/test/README.md [hunk 5]
```diff
@@ -74,3 +74,19 @@ make test                      # Run end-to-end(e2e) tests in a sandbox (virtual
...
   - First, create a Managed CCF instance on your Azure subscription. Please follow [here](https://github.com/microsoft/ccf-app-samples/tree/main/deploy#deploying-the-ccf-samples)
   - Run the e2e-test, please follow [here](https://github.com/microsoft/ccf-app-samples/tree/main/deploy#deploying-a-ccf-application-to-azure-managed-ccf)
+
+## Test JWT tokens
+
+The application currently supports two JWT token issuers (identity providers):
+- **Test Idp:** custom implementation to simulate a token issuer to test your application locally.
+  - *Generate test tokens*: `/.workspace/proposals/set_jwt_issuer_test_sandbox.json` contains pre-generated tokens you can use to test the application endpoints using JWT authentication.
+  - Run ` make start-host` and request `/app/swagger` endpoint then `authorize`
+- **Microsoft Azure Active Directory Identity Provider:** it is an integration sample with MS-AAD Idp
```
Comment (ross-p-smith) [inline data-reconciliation-app/test/README.md:84]:
```suggestion
- **Microsoft Azure Active Directory Identity Provider:** integration sample with MS-AAD Idp
```

**Review by ross-p-smith — APPROVED** [review]

**Pull request merged**